CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/AppShell.xaml

```diff
@@ -72,17 +75,20 @@
     </TabBar>
 
     <TabBar>
-        <ShellContent Title="Asosiy"
-                      ContentTemplate="{DataTemplate views:HomePage}"
-                      Icon="icon_home.png"
-                      Route="HomePage" />
-        <ShellContent Title="Tarix"
-                      ContentTemplate="{DataTemplate views:HistoryPage}"
-                      Icon="icon_history.png"
-                      Route="HistoryPage" />
-        <ShellContent Title="Profil"
-                      ContentTemplate="{DataTemplate views:ProfilePage}"
-                      Icon="icon_user.png"
-                      Route="ProfilePage" />
+        <ShellContent
+            Title="Asosiy"
+            ContentTemplate="{DataTemplate views:HomePage}"
+            Icon="icon_home.png"
+            Route="HomePage" />
+        <ShellContent
+            Title="Tarix"
+            ContentTemplate="{DataTemplate views:HistoryPage}"
+            Icon="icon_history.png"
+            Route="HistoryPage" />
+        <ShellContent
+            Title="Profil"
+            ContentTemplate="{DataTemplate views:ProfilePage}"
+            Icon="icon_user.png"
+            Route="ProfilePage" />
```

Comment:
The same formatting issue, attributes should start on the same level as the Element.